From 6edc9a673a9f79d7f4c5b80b7e49d2c0a54f4ba3 Mon Sep 17 00:00:00 2001
From: Humberto Borba <humberos@gmail.com>
Date: Wed, 9 Aug 2017 12:11:25 -0300
Subject: [PATCH 07/40] base: audioservice: Set BT_SCO status

It is expected from audio HAL during BT SCO routing.

Signed-off-by: Humberto Borba <humberos@gmail.com>
Change-Id: Ifdf7529ff292e9f73fc820400ccc7b6ac7ad112a
---
 services/core/java/com/android/server/audio/AudioService.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/services/core/java/com/android/server/audio/AudioService.java b/services/core/java/com/android/server/audio/AudioService.java
index 570faf4a9a8..f7b139ec0a0 100644
--- a/services/core/java/com/android/server/audio/AudioService.java
+++ b/services/core/java/com/android/server/audio/AudioService.java
@@ -2968,8 +2968,10 @@ public class AudioService extends IAudioService.Stub
                 }
             }
             mForcedUseForComm = AudioSystem.FORCE_BT_SCO;
+            AudioSystem.setParameters("BT_SCO=on");
         } else if (mForcedUseForComm == AudioSystem.FORCE_BT_SCO) {
             mForcedUseForComm = AudioSystem.FORCE_NONE;
+            AudioSystem.setParameters("BT_SCO=off");
         }
         mForcedUseForCommExt = mForcedUseForComm;
         AudioSystem.setParameters("BT_SCO="+ (on ? "on" : "off"));
-- 
2.17.1

